tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,240 @@
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Run list accounts command and verify that it returns list of default accounts
+            await waitForExpect(() => {
+                const stdoutList = execSync(`taq list accounts ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+                expect(stdoutList).toContain("bob");
+            });
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    test('Verify that taqueria flextesa plugin will return "Already running." if sandbox has started" if user tries to call start sandbox twice', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2.  Run start command second time and verify the output
+            const stdoutStart = execSync(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutStart).toEqual("Already running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test('Verify that taqueria flextesa plugin will return "The local sandbox is not running." if user tries to retrieve list of accounts that is not running', async () => {
+        try {
+            // 1. Run list accounts command on sandbox that is not running and verify result
+            const stdoutSandboxIsNotRunning = execSync("taq list accounts local", {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutSandboxIsNotRunning).toEqual("The local sandbox is not running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test('Verify that taqueria flextesa plugin will return "he local sandbox was not running." if user tries to call stop on sandbox that is not running', async () => {
+        try {
+            // 1. Run stop sandbox local on sandbox that is not running and verify result
+            const stdoutSandboxIsNotRunning = execSync("taq stop sandbox local", {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutSandboxIsNotRunning).toEqual("The local sandbox was not running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Currently it cannot be done until this issue has been resolved
+    // https://github.com/ecadlabs/taqueria/issues/243
+    test.skip('Verify that taqueria flextesa plugin can start and stop a sandbox with custom name', async () => {
```

Comment:
Ivan, do you mind also creating an issue for adding "Verify that taqueria flextesa plugin can start and stop a sandbox with custom name" to this test suite?  In the description, you can note that the issue is blocked by #243.

When creating the issue, there are some fields on the left-side of the issue entry screen. If you could set "Project" to "Taqueria", assign to yourself, add the "blocked" label, and then move to the "Blocked" column in the project board, that would be great. Let me know if you have any questions.

Finally, if you could add a link to that new issue in your TODO comment it would be much appreciated. We then won't forgot to come back here when #243 is resolved.